Header header
string node_name
uint64 node_pid
uint16 state_num
exception/SafetyStatus[] status